src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -61,21 +61,28 @@ public function load($path, ...$paths)
     /**
      * Sets values as environment variables (via putenv, $_ENV, and $_SERVER).
      *
-     * Note that existing environment variables are never overridden.
```

Comment:
instead of removing, we should make this accurate, eg 
> Existing environment variables are never overridden, unless they are listed in the SYMFONY_DOTENV_VARS env var.